project_2_image_captioning_project/README.md

```diff
@@ -55,6 +55,8 @@ The decoder model used is the same one from [Show, Attend and Tell: Neural Image
 
    The optimizer used is Adam optimizer. We conclude with the training notebook here and go to the next phase.
 
+- **3_Inference:** The purpose of this file is to make the predictions by loading `trained model` and `vocabulary file` to get the desired result. This model generates good captions for the provided image but it can always be improved later by including hyper-parameters and using more accurate algorithms.  
```

Comment:
Can you replace the title with 3_Inference.ipynb to maintain consistency?

Reply:
@tarunjain1st can you please make this minor change so that I can merge this PR?

Reply:
yeah sure